infra/notifications/src/emailSend.js

```diff
@@ -84,17 +84,20 @@ async function emailSend(
         if (config.verbose) {
           logger.info(`No message found`)
         }
-      } else {
+      } else {        
+        const networkSubDomains = {
+          1: '',
+          4: 'staging.',
+          999: 'localhost.',
+          2222: 'dev.'
+        }
+        const listingNetwork = listing.id.split('-')[0] // First section of id is the network num
         const vars = {
           dappUrl:
-            listing.id[0] === '4'
-              ? 'https://dapp.staging.originprotocol.com'
-              : 'https://dapp.originprotocol.com',
+            `https://dapp.${networkSubDomains[listingNetwork]}originprotocol.com`,
```

Comment:
for network 999, I don't think https://dapp.localhost.originprotocol.com will resolve to local dapp ? I think we use http://localhost on local...

Reply:
Ah crap you are right! Bit the bullet and have everything laid out explicitly.